packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -163,7 +163,7 @@ export interface CaptionsCallFeatureState {
 // @public (undocumented)
 export interface CaptionsInfo {
     captionLanguage?: string;
-    captionText: string;
+    captionText?: string;
```

Comment:
Is there a time we receive a caption without caption text?

Reply:
ACS captions we don't receive caption text because caption language is not supported

Reply:
For ACS captions we display the spoken text and for teams captions we use the caption text